filament/src/details/Scene.h

```diff
@@ -117,7 +117,7 @@ class FScene : public Scene {
     using RenderableSoa = utils::StructureOfArrays<
             utils::EntityInstance<RenderableManager>,   // RENDERABLE_INSTANCE
             math::mat4f,                                // WORLD_TRANSFORM
-            math::mat3f,                                // MATERIAL_ORIENTATION
+            math::quatf,                               // MATERIAL_ORIENTATION
```

Comment:
nit: indentation change is not needed

Reply:
https://github.com/shapr3d/filament/pull/162/commits/6b05fb0008899ce758a2cd8cfc8e1e4b2f094c46